qa/src/test/resources/kapua-datastore-transport-client-setting.properties

```diff
@@ -0,0 +1,32 @@
+###############################################################################
+# Copyright (c) 2011, 2017 Eurotech and/or its affiliates and others
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     Eurotech - initial API and implementation
+#
+###############################################################################
+
+#
+# Elasticsearch Properties
+#NOTE: the datastore client provider implementation must provide:
+#      - a static init method (that initializes the client)
+#      - a static getInstance method (that returns the instance)
+datastore.elasticsearch.client.provider=org.eclipse.kapua.service.datastore.client.embedded.EsEmbeddedClientProvider
+#datastore.elasticsearch.client.provider=org.eclipse.kapua.service.datastore.client.transport.EsTransportClientProvider
+
+#timeouts
+datastore.query.timeout=60000
+datastore.scroll.timeout=60000
+
+datastore.elasticsearch.cluster=kapua-datastore
+datastore.elasticsearch.node=192.168.33.10
```

Comment:
The IP is used with the ES transport client connector and it is supposed to be used with the From the default configuration, the qa and the other tests use the ES embedded node, so this IP is not checked at all during the standard build test phase.
Do you suggest to use another default value (in any case this configuration can be overwritten at startup right?)